github/repos_collaborators.go

```diff
@@ -49,6 +49,35 @@ func (s *RepositoriesService) IsCollaborator(owner, repo, user string) (bool, *R
 	return isCollab, resp, err
 }
 
+// RepositoryPermissionLevel represents the permission level an organization
+// member has for a given repository.
+type RepositoryPermissionLevel struct {
+	// Possible values: "admin", "write", "read", "none"
+	Permission string `json:"permission"`
```

Comment:
What's the reason you decided to use a `string` here instead of `*string`, as is commonly done in other structs?